ref/test/test_output/yppm_0.1.7.test

```diff
@@ -9,7 +9,7 @@ TEST             cry       -0.218562E-02        0.405638E-03       -0.112965E-02
 TEST             dya        0.293576E+04        0.300913E+04        0.298094E+04        0.295948E+04        0.297277E+04
 TEST -------------------------------------------------------------------------------------------------------------------
 TEST ===================================================================================================================
-TEST         Output State
```

Comment:
Why is this formatting wrong now?

Reply:
I added more characters to the text string msg in print_state so the print_state of the original data and the print_state of the interpolated data could be differentiated.  I'm not understanding "formatting wrong".

Here's an example:

TEST ===================================================================================================================
TEST         Input State - Original
TEST ===================================================================================================================
TEST        Variable                 Min                 Max               First                Last                 RMS
TEST -------------------------------------------------------------------------------------------------------------------
TEST             fy2        0.111716E+04        0.111716E+04        0.111716E+04        0.111716E+04        0.111716E+04
TEST               q       -0.255300E-01        0.463537E-01       -0.213277E-02        0.296226E-01        0.167880E-01
TEST             cry        0.694295E-03        0.267743E-02        0.167352E-02        0.173444E-02        0.183947E-02
TEST             dya        0.308891E+04        0.314372E+04        0.312288E+04        0.310786E+04        0.311796E+04
TEST -------------------------------------------------------------------------------------------------------------------


TEST ===================================================================================================================
TEST     Input State - Interpolated
TEST ===================================================================================================================
TEST        Variable                 Min                 Max               First                Last                 RMS
TEST -------------------------------------------------------------------------------------------------------------------
TEST             fy2        0.111716E+04        0.111716E+04        0.111716E+04        0.111716E+04        0.111716E+04
TEST               q       -0.255300E-01        0.463537E-01       -0.213277E-02        0.296226E-01        0.168062E-01
TEST             cry        0.694295E-03        0.267743E-02        0.167352E-02        0.173444E-02        0.184087E-02
TEST             dya        0.308891E+04        0.314372E+04        0.312288E+04        0.310786E+04        0.311798E+04
TEST -------------------------------------------------------------------------------------------------------------------

Reply:
I was thrown off by the right justification of the header text. I was expecting them to be left justified, but I guess they never were.

BTW, to make your paste legible in the comment you have to use triple backticks to preserve the original formatting.  Otherwise, it's just a blob of unreadable text.  You can always use the "preview" button to see what your comment is going to look like before you submit it.  You can close this.  I was just confused by the left/right justification of the headers.